File: backend/src/main/java/luarr/viagemlegal/domain/enums/TipoAnexo.java
package luarr.viagemlegal.domain.enums;

/**
 * Categorias de anexo exigidas conforme o caso.
 * Ver requisitos na página de informações do serviço.
 */
public enum TipoAnexo {
    DOC_REQUERENTE,          // identificação com foto do pai/mãe/guardião
    DOC_MENOR,               // identificação da criança/adolescente
    DOC_ACOMPANHANTE,        // identificação do acompanhante (se houver)
    COMPROVANTE_RESIDENCIA,  // água, telefone ou energia
    PASSAGEM,                // cópia da passagem
    TERMO_GUARDA,            // obrigatório quando requerente é GUARDIAO
    SELFIE_RG                // selfie segurando o RG (fluxo online)
}

File: frontend/src/lib/tipoAnexo.ts
import type { TipoAnexo } from "@/lib/types";

export const TIPO_ANEXO_LABEL: Record<TipoAnexo, string> = {
  DOC_REQUERENTE: "Documento do requerente",
  DOC_MENOR: "Documento da criança/adolescente",
  DOC_ACOMPANHANTE: "Documento do acompanhante",
  COMPROVANTE_RESIDENCIA: "Comprovante de residência",
  PASSAGEM: "Cópia da passagem",
  TERMO_GUARDA: "Termo de guarda/tutela",
  SELFIE_RG: "Selfie segurando o RG",
};
